tests/foreman/cli/test_installer.py

```diff
@@ -17,34 +17,138 @@
 :Upstream: No
 """
 import pytest
-
-pytestmark = [pytest.mark.stubbed]
+from broker import VMBroker
+from broker.hosts import Host
+
+from robottelo.config import settings
+from robottelo.helpers import InstallerCommand
+
+installer_params = {
+    'scenario': "satellite",
+    'foreman-initial-organization': '"Default Organization"',
+    'foreman-initial-location': '"Default Location"',
+    'foreman-initial-admin-username': settings.server.admin_username,
+    'foreman-initial-admin-password': settings.server.admin_password,
+}
+
+
+def host_conf(request):
+    """A function that returns arguments for VMBroker host deployment"""
+    conf = {}
+    conf['workflow'] = "deploy-base-rhel"
+    conf['rhel_version'] = "7.9"
+    return conf
+
+
+def _enable_sat_ports_fw(host):
+    # enable incomming connections to Sat serivces
+    host.execute(
+        'firewall-cmd --permanent --zone=public'
+        ' --add-port 80/tcp --add-port 443/tcp --add-port 5646/tcp'
+        ' --add-port 5647/tcp --add-port 5671/tcp --add-port 8000/tcp'
+        ' --add-port 8140/tcp --add-port 8443/tcp --add-port 9090/tcp'
+        ' --add-port 22/tcp --add-port 2375/tcp --add-port 5000/tcp'
+        ' --add-port 16509/tcp --add-port 53/udp --add-port 69/udp'
+        ' && firewall-cmd --reload'
+    )
+
+
+@pytest.fixture
+def rhel_host_disconnected(request):
+    """A function-level fixture that provides a host which is ready for disconnected installation"""
+    with VMBroker(**host_conf(request), host_classes={'host': Host}) as host:
+        # download RHEL and Satellite ISOs
+        host.execute(f'curl -O {settings.installer.rhel_iso_url}', timeout="7m")
+        host.execute(f'curl -O {settings.installer.sat_iso_url}', timeout="5m")
+        # mount ISOs and configure yum repos
+        host.execute('mkdir /media/rhel7-server')
+        host.execute('mount -o loop $(ls RHEL*.iso) /media/rhel7-server')
+        host.execute('cp /media/rhel7-server/media.repo /etc/yum.repos.d/rhel7-server.repo')
+        host.execute(
+            'echo "baseurl=file:///media/rhel7-server/"' ' >> /etc/yum.repos.d/rhel7-server.repo'
+        )
+        host.execute('')
+        host.execute('mkdir /media/sat6')
+        host.execute('mount -o loop $(ls Satellite*.iso) /media/sat6')
+
+        # temporary solution of too new rhel template
+        # with correct rhel version this should do yum update
+        host.execute(
+            'yum -y downgrade '
+            ' grub2 grub2-common grub2-pc grub2-pc-modules'
+            ' grub2-tools grub2-tools-extra grub2-tools-minimal'
+            ' glibc glibc-common'
+            ' rpm rpm-libs rpm-python rpm-build-libs'
+            ' libmount krb5-libs libuuid libblkid util-linux libsmartcols'
+            ' nss nss-softokn nss-sysinit nss-tools nss-softokn-freebl'
+            ' python python-libs'
+        )
+        _enable_sat_ports_fw(host)
+
+        yield host
+
+
+@pytest.fixture
+def rhel_host_connected(request):
```

Comment:
RHEL7/RHEL8 parametrization + config values related to the particular RHEL version